sql/core/src/test/resources/sql-tests/inputs/transform.sql

```diff
@@ -0,0 +1,47 @@
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW t1 AS SELECT * FROM VALUES
+('a'), ('b'), ('v')
+as t1(a);
+
+CREATE OR REPLACE TEMPORARY VIEW t2 AS SELECT * FROM VALUES
+(1, '1', 1.0, Decimal(1.0), timestamp(1)),
+(2, '2', 2.0, Decimal(2.0), timestamp(2)),
+(2, '2', 3.0, Decimal(3.0), timestamp(3))
```

Comment:
Could you add all the typed data for tests where possible, e.g., `date` and `float`.

Reply:
> Could you add all the typed data for tests where possible, e.g., `date` and `float`.

Updated.